index.html

```diff
@@ -103,6 +104,7 @@
         <a href="/learn">Learn</a>
         <a href="#events">Events</a>
         <a href="https://databasediisc.blogspot.com/">Blog</a>
+        <a href="/pages/projects" class="nav-list-link">Projects</a>
```

Comment:
This is good for now, but I will get back to you with a confirmation on whether or not this should be on the navbar. What is your opinion on it?

Reply:
Navbar seems good. But having a proper section on the home page might be better.

Reply:
Would be better to have a projects section. Because (as promised), we need to host the delta projects on our website(at least the completed ones).
I would advise to have one though!

Reply:
Cool, then let's keep it. @TheUltimateKeval Can you open another PR on this repository to remove the link to "Learn" (#16)? We can then revert the change in the Navbar dimensions. (for now)

Reply:
Instead of a new PR, I will add a commit to this PR which resolves #16.

Reply:
> Would be better to have a projects section. Because (as promised), we need to host the delta projects on our website(at least the completed ones). I would advise to have one though!

Just to be clear, the project section on the home page would only showcase 1 or 2 projects and will have a button which leads to the dedicated projects page which has been added.

Reply:
It's best to keep PRs atomic 😃 
Great work again! Let me review and approve.

Reply:
Will try to adhere to that principle going forward, but my concern was dealing with unnecessary merge conflicts (not very experienced with handling merges yet).

For now, I request you to merge the commit addressing #16 with this PR after I add one or two projects to the projects page. I think we can then merge this PR.

Since you conveyed earlier that we need not worry about having individual pages for projects on this website, we can defer the second task until we have the need for working on it.

Reply:
Sure, its okay! This PR is fine; we will merge it the way it is.